### ISSUE TITLE
Add support for Item chat descriptions

### ISSUE THREAD

**Issue by stanavdb** [the-metalworks/cosmere-rpg#141]
**Is your feature request related to a problem? Please describe.**  
Some items need to have long descriptions, whose content isn't necessarily all relevant for the chat, but it gets added to the chat card anyway. This causes items which don't have a roll as part of their activation to put an incredibly long expanded message into the chat.

**Describe the solution you'd like**  
Add support for alternate chat descriptions that are used in the chat message instead of the proper item description.

**Describe alternatives you've considered**  
-

**Version**  
0.1.1

**Checklist:**
- [x] I have searched the existing issues to ensure this feature has not already been requested.

**Comment by zithith** [comment]
There is already "back-end" functionality to support different tiers of description (full/short/chat) this will effectively just need to provide a UI in the item description template for editing those, then passing through the chat option as the item's description when activated if it has one, otherwise fall back to short and then full.
@MangoFVTT do we think this falling back should be handled by the caller, or the chat message? We can just hand the whole description object into the message instead?

**Comment by MangoFVTT** [comment]
>@MangoFVTT do we think this falling back should be handled by the caller, or the chat message? We can just hand the whole description object into the message instead?

As it currently works, the item generates the description HTML before ever reaching the chat message stage and just passes the entire HTML object containing the description to the chat message so it can be properly enriched. It should be relatively trivial to change what text is placed within that description HTML

**Comment by zithith** [comment]
Fair enough, no need to change this up, that will get handled in the item
then 👍🏻

**Comment by LLilliputian** [comment]
Hey just to mention am taking a look at this, have mentioned to Zithith on their PR. Sorry again for not posting here previously!

**Comment by zithith** [comment]
@LLilliputian how are you getting on with this? Are we ready for PR?

**Comment by LLilliputian** [comment]
Hey sorry missed this message till now. Work slowed a little cause of my job starting up again but am mostly there now I think.
I've gotten it set up that the description tab now has three sections for the main description, the short description, and the chat description that are each able to collapse.
I've been trying to set it up so that when you go to edit them it opens a prose mirror as the full tab but have been having issues having it create the prose mirror in edit mode and also returning back to the original description when its saved so still looking into that.

I can set up a PR later today in draft so can at least see where its at.

**Comment by LLilliputian** [comment]
Hey, managed to figure out the prose mirror stuff so have undrafted the PR.

Think the CSS stuff is likely to need a bit of a look at as I've not done much of it before and am not great at it. Functionally should all work though I think 🤞
